Sources/Sentry/SentryClient.m

```diff
@@ -794,6 +794,7 @@ - (void)applyExtraDeviceContextToEvent:(SentryEvent *)event
                 block:^(NSMutableDictionary *device) {
                     device[SentryDeviceContextFreeMemoryKey] = @(self.crashWrapper.freeMemorySize);
                     device[@"free_storage"] = @(self.crashWrapper.freeStorageSize);
+                    device[@"processor_count"] = @(NSProcessInfo.processInfo.processorCount);
```

Comment:
`m`: Please use `SentryNSProcessInfoWrapper.processorCount` instead and adapt the tests accordingly.